### PR TITLE
Metric chart style updates

### PR DESCRIPTION
**Pull request by chasulin**
- Fixes width of ICU tooltip
- Drops `marginLeft` to allow chart to take up maximum space on mobile
- Drops chart line stroke-width to 2px on mobile (vs. 4px on desktop)
- Drops chart height to 280px on mobile (vs. 400 px on desktop)

### CODE REVIEW

File: src/components/Charts/BoxedAnnotation.tsx [hunk 1]
```diff
@@ -33,6 +33,8 @@ export const BoxedAnnotation = ({
         x={left - padding}
         width={Math.abs(width + 2 * padding)}
         height={Math.abs(height + 2 * padding)}
+        rx={3}
+        ry={3}
```
Comment (pnavarrc) [inline src/components/Charts/BoxedAnnotation.tsx:37]: **Note**: According to [CSS Tricks](https://css-tricks.com/svg-properties-and-css/), we should be able to set`rx` and `ry` in CSS, but they are ignored in Chrome iOS. Adding them as attributes solves the issue.

File: src/components/Charts/ChartICUCapacityUsed.tsx [hunk 2]
```diff
@@ -13,12 +13,12 @@ const getTooltipContent = (valueY: number) =>
     ? {
         subtitle: `ICU capacity used >`,
         body: `${getPointText(CAP_Y)}`,
-        width: 'auto',
+        width: '125px',
```
Comment (chasulin) [inline src/components/Charts/ChartICUCapacityUsed.tsx:16]: matches tooltip width in `ChartPositiveTestRate.tsx`

**Review by pnavarrc — APPROVED** [review]
🙌  looks good!

**Comment by pnavarrc** [comment]
[Make charts more legible on mobile (try making them shorter, t-hold labels on the left)](https://trello.com/c/wunCrzeG/930-make-charts-more-legible-on-mobile-try-making-them-shorter-t-hold-labels-on-the-left)

**Pull request merged**